Userland/Applications/FontEditor/FontEditor.h

```diff
@@ -15,7 +16,8 @@
 class GlyphEditorWidget;
 class GlyphMapWidget;
 
-class FontEditorWidget final : public GUI::Widget {
+class FontEditorWidget final : public GUI::Widget
+    , public Config::Listener {
```

Comment:
Please format this as

```suggestion
class FontEditorWidget final
    : public GUI::Widget
    , public Config::Listener {
```

Reply:
Fixed.